sdk/data/aztables/README.md

```diff
@@ -1,151 +1,342 @@
-
-Common uses of the Table service include:
-
-- Storing TBs of structured data capable of serving web scale applications
-- Storing datasets that don't require complex joins, foreign keys, or stored procedures and can be de-normalized for fast access
-- Quickly querying data using a clustered index
-### Create the Table service client
-
-First, we need to construct a `TableServiceClient`.
-
-### Create an Azure table
-Next, we can create a new table.
-
-
-### Get an Azure table
-The set of existing Azure tables can be queries using an OData filter.
-
-
-### Delete an Azure table
-
-Individual tables can be deleted from the service.
-
-
-### Create the Table client
-
-To interact with table entities, we must first construct a `TableClient`.
-
-
-### Add table entities
-
-Let's define a new `TableEntity` so that we can add it to the table.
+## Troubleshooting
 
-Using the `TableClient` we can now add our new entity to the table.
+### Error Handling
 
+All I/O operations will return an `error` that can be investigated to discover more information about the error. In addition, you can investigate the raw response of any response object:
+```golang
+resp, err := client.CreateTable(context.Background(), nil)
+if err != nil {
+    err = errors.As(err, azcore.HTTPResponse)
+    // handle err ...
+}
+```
 
-### Query table entities
+### Logging
 
-To inspect the set of existing table entities, we can query the table using an OData filter.
+This module uses the classification based logging implementation in azcore. To turn on logging set `AZURE_SDK_GO_LOGGING` to `all`. If you only want to include logs for `aztables`, you must create your own logger and set the log classification as `LogCredential`.
 
+To obtain more detailed logging, including request/response bodies and header values, make sure to leave the logger as default or enable the `LogRequest` and/or `LogResponse` classificatons. A logger that only includes credential logs can be like the following:
 
-If you prefer LINQ style query expressions, we can query the table using that syntax as well.
+```go
+import azlog "github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
+// Set log to output to the console
+log.SetListener(func(cls log.Classification, msg string) {
+		fmt.Println(msg) // printing log out to the console
+})
 
+// Includes only requests and responses in credential logs
+log.SetClassifications(log.Request, log.Response)
+```
 
-### Delete table entities
+> CAUTION: logs from credentials contain sensitive information.
+> These logs must be protected to avoid compromising account security.
 
-If we no longer need our new table entity, it can be deleted.
+## Next steps
 
+## Provide Feedback
 
-## Troubleshooting
+If you encounter bugs or have suggestions, please
+[open an issue](https://github.com/Azure/azure-sdk-for-go/issues) and assign the `Azure.Identity` label.
```

Comment:
Not 'Azure.Identity', right? Maybe 'Azure.Tables' or similar for the label?

Reply:
good catch